src/bin/ambit/directories.rs

```diff
@@ -95,11 +96,12 @@ impl AmbitPaths {
         let home = dirs::home_dir().expect("Could not get home directory");
         let configuration = home.join(".config/ambit");
 
-        let config = AmbitPath::new(configuration.join("config.ambit"), AmbitPathKind::File);
-        let repo = AmbitPath::new(configuration.join("repo"), AmbitPathKind::Directory);
-        let git = AmbitPath::new(configuration.join("repo/.git"), AmbitPathKind::Directory);
-
-        AmbitPaths { config, repo, git }
+        AmbitPaths {
+            home: AmbitPath::new(home, AmbitPathKind::Directory),
+            config: AmbitPath::new(configuration.join("config.ambit"), AmbitPathKind::File),
+            repo: AmbitPath::new(configuration.join("repo"), AmbitPathKind::Directory),
+            git: AmbitPath::new(configuration.join("repo/.git"), AmbitPathKind::Directory),
```

Comment:
This should probably be `configuration.join("repo").join(".git")`, for fear of upsetting Windows users.